app/models/result.rb

```diff
@@ -0,0 +1,25 @@
+class Result
```

Comment:
It's useful to track whether pretty much anything was successful or not! I'm just using it here for that reason.

It lets you return the truthy/falsey "was this a success?" while also giving back either errors or the result of whatever you just did.

(also shamelessly stole this from @tarebyte because I loved the idea of it when I saw it in Classroom)